ansible/roles/gpu/templates/config.toml.tmpl

```diff
@@ -0,0 +1,157 @@
+version = 2
```

Comment:
Do we really need to overwrite the whole `config.toml` or can we use separate config files just to override GPU stuff (this should be supported by containerd ootb)?

Reply:
No, we dont - the default one does it (is identical by now).